src/components/posters/Posters.js

```diff
@@ -15,13 +16,14 @@ const Posters = ({movies, selectMovie}) => {
   if (movies) {
     posterCards = movies.map(movie => {
       return (
-        <Poster
-          key={movie.id}
-          id={movie.id}
-          photo={movie['poster_path']}
-          title={movie.title}
-          selectMovie={selectMovie}
-        />
+        <Link to={`${movie.id}`} key={movie.id}>
```

Comment:
Should this be `to={`/${movie.id}`}` ?

Reply:
Doesn't seem to matter either way. Same behavior. Not sure if there is a best practice.